fili-generic-example/README.md

```diff
@@ -58,26 +67,37 @@ Here are some sample queries that you can run to verify your server:
 
 ### Specific to Wikipedia data
```

Comment:
We probably want to move the stuff you added here into a separate section because the _wikipedia_ data in the docker conatiner is different from the _wikipedia_ data in the [druid quickstart tutorial](http://druid.io/docs/latest/tutorials/quickstart.html)

It might also be better suited in [fili-wikipedia-example](https://github.com/yahoo/fili/tree/master/fili-wikipedia-example) at the top as a quickstart guide, but we should probably still mention that the data is different.
@michael-mclawhorn ?